src/components/search/Search.js

```diff
@@ -94,16 +95,43 @@ const Search = (props) => {
     setValue(e.target.value)
   }
 
-  const handleSelect = async (description) => {
-    setValue(description, false)
+  const isValidCoordinate = () => {
+    try {
+      return new CoordinateParser(value)
+    } catch {
+      return false
+    }
+  }
 
-    dispatch(
-      searchView(
-        await getPlaceBounds(descriptionToPlaceId.current[description]),
-      ),
+  const handleCoordinateSearch = () => {
+    const coordinate = new CoordinateParser(value)
+
+    return (
+      <ComboboxOption
+        as={SearchEntry}
+        key={`${coordinate.getLatitude()}, ${coordinate.getLongitude()}`}
+        value={`${coordinate.getLatitude()}, ${coordinate.getLongitude()}`}
```

Comment:
You can define a variable for this string, and just use it a couple times.

Reply:
You can make this a function `getCoordinatesResult()`, and just return null if the coordinate parser returns an error.